packages/react/src/components/Popover/Popover.mdx

```diff
@@ -19,13 +19,53 @@ import { Preview, Props, Story } from '@storybook/addon-docs/blocks';
 
 **Note: this component is currently experimental and does not follow semver.**
 
-The Popover component is used to build layers that appear on top of the current
-surface.
+A popover is a UI element that _pops over_ or renders on top of other UI on the
+page. It's a component "primitive" meaning it can be used to help make more
+complex components that utilize this behavior. It is often styled as what we'd
+traditionally consider a tooltip.
+
+## Usage 
+
+```jsx
+import { popover } from 'carbon-components-react';
+
+function MyPopover() {
+  return (
+    <TriggeringUiElement/>
+    <Popover 
+      open={open} 
+      align={top} 
+      relative>
```

Comment:
```suggestion
     >
```

We might need to figure out what to do with this `relative` thing since I think we need the container to be relative in this case 🤔